src/common/modals/ShowScreenShot.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import path from "path";
+import styled from "styled-components";
+import Modal from "../components/Modal";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../components/Modal"](https://app.codacy.com/manual/gorilla-devs/GDLauncher/pullRequest?prid=5082094)